lib/routemaster/cache.rb

```diff
@@ -102,7 +63,13 @@ def get(url, version: nil, locale: nil)
     # @return [FutureResponse], which responds to `status`, `headers`, and `body`
     # like [Response].
     def fget(url, version: nil, locale: nil)
-      FutureResponse.new { get(url, version: version, locale: locale) }
+      headers = {
+        'Accept' => version ?
+          "application/json;v=#{version}" :
+          "application/json"
+      }
+      headers['Accept-Language'] = locale if locale
```

Comment:
this `headers` setting bit is identical in the `#get` method, which means that it could possibly be refactored into its own method :)